numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -240,6 +240,15 @@ gentype_@name@(PyObject *m1, PyObject *m2)
 /**end repeat**/
 #endif
 
+#if PY_VERSION_HEX >= 0x03050000
+static PyObject *
+gentype21_not_implemented(PyObject *m1, PyObject *m2)
```

Comment:
21?

Reply:
two inputs, one output

Reply:
I think it's clearer just to call these `gentype_matmul` and whatever the other name is.

Alternatively - can you just leave the members as null to get this behavior?

Reply:
yes, removing